doc/content/applications/se-vanilla/downlinks.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "downlinks"
+description: ""
+weight: 2
+---
+
+The vanilla app can be configured using downlinks to control the device functionality and effectively unlock new features.
+
+Each device component is controlled on a separate FPort, but please remember that LoRaWAN class A devices can only receive a downlink when they transmit and uplink.
+
+So make sure you trigger an uplink via the button or wait for a periodic uplink message in order for the device to receive your downlink.
+
+## Heartbeat
```

Comment:
A payload downlink formatter may be added later for this, but the commands are small enough to be doable using binary/hex.

Reply:
Yes indeed, I leave this up to you guys to play with and add to this documentation in the future